### DISCUSSION TITLE
Skipped entity message when calling "draw_layout" function part of the FRONTEND module on the drawing add on

### DISCUSSION THREAD

**Discussion by Jaime-Moniet** [mozman/ezdxf#631] [Q&A] [answered]
Hi,

I was wondering if there are any tips on how to avoid the skipped entity message when calling the `draw_layout` function part of the `FRONTEND` module on the drawing add on. I get hundreds of lines of the same skipped entity message when running my script, but the script runs correctly at the end and creates all the PDFand SVG files correctly.

message: "skipped entity ATTRIB(#None). Reason: "invisible"" 

I was exploring the `filter_func` argument in the `draw_layout` function to skip any of the invisible entities and bypass the log_message but I have not been successful.

**Comment by mozman** [comment, marked as answer]
Create your own Frontend by inheriting from `ezdxf.adddons.drawing.frontend.Frontend` and override the `skip_entity()` method.

**Reply by mbway** [reply]
I agree that a custom frontend is a good idea but now that ezdxf uses `logging` extensively, perhaps the default `log_message()` should output to the ezdxf logger so it could also be silenced by setting the level of that logger? (I know I was the one to originally put the `print()` in there so sorry about that).
I think it's still important to have `log_message` though and not just send the message to the ezdxf logger from `skip_entity()` in case the user wants to do something special in a custom frontend.

**Reply by mozman** [reply]
Replaced `print()` by `logger.info()` in `Frontend.log_message()`.

**Reply by Jaime-Moniet** [reply]
Thank you @mozman and @mbway ! This solution works for me.
